package/CHANGELOG

```diff
@@ -19,6 +19,7 @@ The rules for this file:
   * 0.16.0
 
 Enhancements
+  * Added unit tests for bad file mode passed to SelectionWriter
```

Comment:
@jbarnoud and @fiona-naughton made extensive use of the testsuite CHANGELOG for this purpose.